docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc

```diff
@@ -0,0 +1,180 @@
+
+* work item handlers
+* event listeners
+* marshalling strategies
+* globals
+* environment entries
+* configuration
+Validation is performed automatically when a build is invoked or manually when user clicks on Validate button.
+
+=== Error Handling
+.New error handling
+The system now detects and stores execution errors, such as exceptions while running a process instance or a job after the maximum retry number has been exceeded. 
+
+.New Execution Errors View
+A new view to explore and acknowledge the new generated execution error has been added.
+
+.Configurable Error Handling
+If an error occurs, an event is generated that contains the following data:
+
+--
+[cols="1,2",options="header"]
+|===
+| Data
+| Fields
+
+.3+| Process model
+| processModelID
+| processModelVersion
+| processModelName
+
+.2+| Process instance that has failed
+| processInstanceID
+| customProcessID
+
+.5+| Activity that has failed
+| activityID
+| activityName
+| activityType
+| iteration
+| status
+
+.2+| Error data
+| errorMessage: Message with the error that has occurred.
+| errorStack: Stack with error detail.
+|===
+--
+
+== Process Engine
+=== Process and task administration API
+
+A process administration API and task administration API have been introduced to simplify some of the more complex administrator use cases.
+
+=== Advanced task routing
+When tasks are assigned to a group of users, you can use pluggable task assignment strategies to automatically assign tasks to a suitable individual immediately (when the task is being created or later released for example). This enables more efficient task allocation, based on all properties associated with the task (for example potential owners and task priority but also task data that includes information such as geography, required skills, and so forth). You can use business rules to define the assignment logic, making it easy to customize this to your needs.
+
+=== Process Execution Server
+The process execution server (also known as kie-server) has been extended to support core engine features and offers a remote API for these operations. 
+
+=== Separate workbench from execution server
+The workbench now delegates all of its requests to the execution server as well. The main advantage is that the workbench can now be used to monitor any set of execution servers. By linking the execution server to the workbench, the process and task monitoring UIs in the workbench can now connect to this execution server and show all relevant information. When multiple independent execution servers are used, you can either connect to a specific one or use the smart router to aggregate information across multiple servers. 
+
+=== Smart router
+The smart router (also known as kie-server-router) can be used as a proxy to help manage multiple independent process execution servers.
+ 
+=== KIE Server Router enhancements
```

Comment:
KIE Server Router is same component as Smart router, so this point could be merged with previous one. Considering that Smart router is a new component in 7.x I think this point could be skipped completely.

Reply:
@sutaakar removed section